packages/venia-concept/src/components/ProductOptions/swatch.js

```diff
@@ -68,16 +67,13 @@ class Swatch extends Component {
         });
 
         return (
-            <Tooltip text={label}>
-                <button
-                    {...restProps}
-                    className={className}
-                    style={finalStyle}
-                    title={label}
```

Comment:
Any reason why you removed the `title` prop?

Reply:
Just to avoid default tooltip generated created by the title attribute.
I will disable with css if it affects a11y

Reply:
That default tooltip from the `title` attribute is very important for accessibility, even for users without vision impairment. I do suggest restoring it (and not hiding it with CSS).

Reply:
It doesn't affect this context (button with only an icon as child content), but I read somewhere that this was an accessibility anti-pattern:

`<button title="Thing">Thing</button>`

Which a screen reader would announce as: 

`BUTTON Thing TITLE Thing`

Has anyone encountered that, or has the practice changed? Years ago a screenreader user was begging people to stop putting redundant title tags on everything because it clogged the audio.

Reply:
sure will restore @jimbo @brendanfalkowski